test/verify/check-shell-keys

```diff
@@ -105,7 +105,22 @@ class TestKeys(MachineCase):
         # Log in as admin
         login("admin", "foobar", "Administrator")
         b.go("#/user")
-        b.wait_in_text("#account-authorized-keys-list li.pf-c-data-list__item:first-child", "no authorized public keys")
+        b.wait_in_text("#account-authorized-keys-list li.pf-c-data-list__item:first-child",
+                       "You do not have permission to view the authorized public keys for this account.")
+
+        # Get amdin privs
```

Comment:
"amdin" typo. As you need to push anyway, would be nice to fix that, as we often grep the code for user names.

Reply:
Fixed.